### DISCUSSION TITLE
RPM & openSUSE Repository

### DISCUSSION THREAD

**Discussion by rickykresslein** [espanso/espanso#1209] [Feature requests and Ideas]
I've created a repository on openSUSE's Build Service for espanso. Anyone on openSUSE Tumbleweed can now install espanso with the following commands:

`zypper addrepo https://download.opensuse.org/repositories/home:rickyk/openSUSE_Tumbleweed/home:rickyk.repo`
`zypper refresh`
`zypper install espanso`

Those on openSUSE Leap 15.2 can do the following:

`zypper addrepo https://download.opensuse.org/repositories/home:rickyk/openSUSE_Leap_15.2/home:rickyk.repo`
`zypper refresh`
`zypper install espanso`

There is an RPM available on the openSUSE repo page [here](https://software.opensuse.org//download.html?project=home%3Arickyk&package=espanso).

Please let me know if you have any issues so I can get them worked out.

**Comment by federico-terzi** [comment]
Hey @rickykresslein,

Thanks for your help! After #594 is ready, I'll be happy to collaborate to create an official RPM release :)

Cheers :)

**Comment by pointydev** [comment]
Would love to see this come to fruition with the new pre-release out now.

**Comment by AucaCoyan** [comment]
Closing in favour of the meta issue #1948
